examples/js/effects/AnaglyphEffect.js

```diff
@@ -2,10 +2,31 @@
  * @author mrdoob / http://mrdoob.com/
  * @author marklundin / http://mark-lundin.com/
  * @author alteredq / http://alteredqualia.com/
+ * @author tschw
  */
 
 THREE.AnaglyphEffect = function ( renderer, width, height ) {
 
+	// Matrices generated with angler.js https://github.com/tschw/angler.js/
+	// (in column-major element order, as accepted by WebGL)
+
+	this.colorMatrixLeft = new THREE.Matrix3().set(
```

Comment:
Yes. More precisely, it's why I kept them public - the code was written a while ago.

Historically [my library](https://github.com/tschw/angler.js/) started as Three.js example code. It computes these matrices from various inputs, such as e.g. the combination of color filters (there are green/magenta and yellow/blue glasses out there) and translucency balance, similar to the well-known "Balance" control of stereo audio systems (the most significant difference between different consumer-grade 3D glasses with the same filter colors is the relative translucency between the two filters, according to the spectral curves provided by [Woods et al](http://cmst.curtin.edu.au/local/docs/pubs/2010-11.pdf)).  

Because of its applicability to photo and video, I ended up writing the WebGL code and anything else from scratch. I considered using the math-only build of Three.js for the Shadertoy player, but then opted for self-containment using a bunch of [array-based, assembly-like math routines](https://github.com/tschw/angler.js/blob/master/app/classes/AnaglyphShadertoyPlayer.js#L543-L611)  for computing camera quaternion & position, which was all I would have needed it for, since the stereo stuff / ray generator lives in a GLSL vertex shader.

Adding a default and a link on how it was calculated seemed reasonable to both keep the Three.js example exemplary and to pave the way towards more control. If you like, we can add the minified lib and equip the example with a GUI to select glasses and stuff at some point in the future. Right now, it still takes tweaking of many parameters to achieve optimal results, so I'd prefer to do that once we can get away with a lean GUI where every input is immediately obvious to the user.